AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/Model/KeyStore.smithy

```diff
@@ -146,6 +146,9 @@ structure GetKeyStoreInfoOutput {
   @javadoc("An identifier for this Key Store.")
   keyStoreId: String,
   @required
+  @javadoc("An identifier to be used in the cache for this Key Store.")
+  keyStoreCacheId: String,
```

Comment:
I think this is a better name.
```suggestion
  binaryId: Blob,
```

Reply:
`binaryId` is already being used elsewhere in the code for something else [[ref](https://github.com/aws/aws-cryptographic-material-providers-library/blob/8c8a38ff3534b8e4f0dca020e65effc454ef330a/AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/AlgorithmSuites.dfy#L130)]

Reply:
So is Key Store. 
It's ok to repeat names; especially when they will never be referenced in the same context.